pkg/kubectl/scale.go

```diff
@@ -139,6 +145,9 @@ func (scaler *ReplicationControllerScaler) ScaleSimple(namespace, name string, p
 	controller.Spec.Replicas = int(newSize)
 	// TODO: do retry on 409 errors here?
 	if _, err := scaler.c.UpdateReplicationController(namespace, controller); err != nil {
+		if errors.IsInvalid(err) {
```

Comment:
@derekwaynecarr do admission rejections because of quota show up as IsInvalid errors? Are there any Invalid errors that could go away after a time? If so, how do we ensure the RC retries after a time?